doc/source/whatsnew/v0.24.1.rst

```diff
@@ -0,0 +1,77 @@
+:orphan:
```

Comment:
This orphan tag is so that sphinx doesn't warn that it isn't included in a toctree.

After 0.24.0 is released, we will

- Update index.rst.template to change 0.24.0 to 0.24.1
- Remove this orphan tag